python/hail/table.py

```diff
@@ -2027,6 +2028,8 @@ def explode(self, field):
         ----------
         field : :obj:`str` or :class:`.Expression`
             Top-level field name or expression.
+        name : :obj:`str` or None
+            If not `None`, name for the exploded field.
```

Comment:
This could probably use a bit more documentation. In the existing example, you could have it explode `Children` into `Child`. But it should also state that the default behavior is to reuse the old name.